src/components/Welcome.js

```diff
@@ -52,7 +52,7 @@ const Welcome = ({ firestore }) => {
     const newToken = GetToken();
     saveToken(newToken);
     setToken(newToken);
-    push('/addItem');
+    push('/list');
```

Comment:
Yes! I love that you're sending them through the list first and then to the addItem view from there. Sending the user through a nice flow like this is training them how to use the tool. 👏